lib/db/schema/patch-022-023.sql

```diff
@@ -0,0 +1,133 @@
+-- This migration:
+--
+-- * Changes the type of the devices.callbackPublicKey column
+-- * Add a new devices.callbackAuthKey column
+-- * Updates the accountDevices stored procedure with the new column
+-- * Alters three stored procedures, createDevice, updateDevice and
+--   sessionWithDevice to work with the new columns
+
+ALTER TABLE devices
+MODIFY COLUMN callbackPublicKey VARCHAR(88),
+ADD COLUMN callbackAuthKey VARCHAR(24);
```

Comment:
With respect to the possibility of junk data in the existing column, it may be worth leading with a deployment of something like https://github.com/mozilla/fxa-auth-server/pull/1235, explicitly disabling the auth-server from storing keys until we have the bits in place to handle them.